.testcaferc.json

```diff
@@ -1,4 +1,5 @@
 {
+  "appCommand": "npm run serve",
```

Comment:
Q: how is this command run? I did not see it explicitly called in the `test:e2e` script

Reply:
`test:e2e` calls `testcafe`, which looks at this config file (and runs this command) 🕸

Reply:
ah ok, thanks for clarifying. i was looking at the command line docs, but finally found this `https://devexpress.github.io/testcafe/documentation/using-testcafe/configuration-file.html#appcommand`